extensions/amp-analytics/0.1/test/test-amp-analytics.js

```diff
@@ -798,20 +798,6 @@ describes.realWin(
         ':not(p)',
         'p:nth-child(2)',
       ].map(selectorExpansionTest);
-
-      it('does not expands selector with platform variable', () => {
```

Comment:
Is there a reason we don't want to use the platform macros here? We could add something like a `noPlatform` flag for certain callers if you like. This would come with the cost that these callers could not use nested.

Reply:
This behavior has changed, let me know if you think we need to support not expanding macros in certain cases.

Reply:
I guess it's fine. Please add a note to the code. Thanks